src/ol/renderer/canvas/VectorImageLayer.js

```diff
@@ -107,7 +107,7 @@ class CanvasVectorImageLayerRenderer extends CanvasImageLayerRenderer {
       vectorRenderer.useContainer(null, null);
       const context = vectorRenderer.context;
       const layerState = frameState.layerStatesArray[frameState.layerIndex];
-      context.globalAlpha = layerState.opacity;
+      this.globalAlpha = layerState.opacity;
```

Comment:
This change essentially means that no layer opacity will be applied at all, because there is no `globalAlpha` member property. Removing the line would have the same effect, and this would result in the same opacity behavior that we have on `ol/layer/Vector`. Is that your intention?